### PR TITLE
Add sprinting

### PR DESCRIPTION
**Pull request by Hedreon**
You can revert the version number or correct it.

Full description:
Add sprinting to add more realism + I changed the version number for whatever reason. The maintainers are also able to...
> _revert the version number or correct it._

### CODE REVIEW

File: RealismClient/Sprint.client.lua [hunk 1]
```diff
@@ -0,0 +1,26 @@
+script.Parent = game.StarterPlayer.StarterCharacterScripts
+
+local uis = game:GetService("UserInputService")
+local ts = game:GetService("TweenService")
+local ti = TweenInfo.new(0.25, Enum.EasingStyle.Sine, Enum.EasingDirection.InOut, 0, false, 0)
+local plr = game.Players.LocalPlayer
+local char = plr.Character or plr.CharacterAdded:Wait()
+local cam = workspace.CurrentCamera
```
Comment (Phalanxia) [inline RealismClient/Sprint.client.lua:8]: I suggest not using shorthand variables in this. See this file as an example:
https://github.com/CloneTrooper1019/Character-Realism/blob/main/RealismClient/init.client.lua#L8-L12

Comment (Hedreon) [reply RealismClient/Sprint.client.lua:8]: okay, I'll reformat it.

Comment (Hedreon) [reply RealismClient/Sprint.client.lua]: Reformated, hopefully it fits the format now.

**Comment by Hedreon** [comment]
I have to re-open PR due to conflicts.

**Pull request closed without merging**